tests/ui/manual_float_methods.rs

```diff
@@ -40,7 +40,6 @@ fn main() {
     // Not -inf
     if x != f64::INFINITY && x != fn_test_not_inf() {}
     const X: f64 = 1.0f64;
-    // Will be linted if `const_float_classify` is enabled
     if const { X == f64::INFINITY || X == f64::NEG_INFINITY } {}
```

Comment:
Would be good to have a test where this now emits a warning. Something like
```rs
const {
  let x = 1.0f64;
  if x == f64::INFINITY || x == f64::NEG_INFINITY {}
}
```
should suffice I think